task_processing/plugins/kubernetes/types.py

```diff
@@ -27,6 +16,22 @@ class SecretEnvSource(TypedDict):
     key: str
 
 
+class NodeAffinityOperator:
+    IN = "In"
+    NOT_IN = "NotIn"
+    EXISTS = "Exists"
+    DOES_NOT_EXIST = "DoesNotExist"
+    GT = "Gt"
+    LT = "Lt"
+    ALL = {IN, NOT_IN, EXISTS, DOES_NOT_EXIST, GT, LT}
+
+
+class NodeAffinity(TypedDict):
+    key: str
+    operator: str
+    value: Any
```

Comment:
just curious: do the k8s docs mention what values are expected? although, i guess that even if they do we'd have a pretty ugly Union here instead of an Any

Reply:
[k8s docs](https://github.com/kubernetes-client/python/blob/master/kubernetes/docs/V1NodeSelectorRequirement.md) ask for list[str], with additional reqs for the diff operators.

Reply:
yea, leaving this as Any seems fine then